src/Types.ts

```diff
@@ -145,14 +131,13 @@ export interface FocusedElementDetail {
     modalizerId?: string;
 }
 
-export const AsyncFocusSources = {
-    EscapeGroupper: 1,
-    Restorer: 2,
-    Deloser: 3,
-} as const;
+export interface AsyncFocusSources {
+    EscapeGroupper: 1;
+    Restorer: 2;
+    Deloser: 3;
+}
 
-export type AsyncFocusSource =
-    typeof AsyncFocusSources[keyof typeof AsyncFocusSources];
+export type AsyncFocusSource = AsyncFocusSources[keyof AsyncFocusSources];
```

Comment:
why is this type necessary, I have seen it used nowhere

Reply:
can this be removed without breaking the API?

Reply:
It is used just a few lines below in `requestAsyncFocus()` and `cancelAsyncFocus()` methods. Basically all of these singular types are for the enum-like pattern completeness. Singular type is for the variables and props of the enum type, plural hold possible values. `function goodFunc(prop: SingularType)` and `goodFunc(PluralType.Option1)`. With native enums it is the same name for both typing the props and picking values, but we have opted to not use native enums because of the nasty way they are transpiled.